### PR TITLE
[TASK] Generate declare(strict_types=1) for all PHP files

### PR DESCRIPTION
**Pull request by liayn**
Resolves: #53

### CODE REVIEW

**Comment by simonschaufi** [comment]
Looks good but failing tests

File: Classes/Parser/NodeFactory.php [hunk 1]
```diff
@@ -106,12 +106,12 @@ public function buildClassNode($classObject, $skipStatements = false)
...
 
     /**
-     * @param \EBT\ExtensionBuilder\Domain\Model\File $fileObject
+     * @param File $fileObject
      * @return array
      */
     public function getFileStatements(File $fileObject)
     {
-        $stmts = [];
+        $stmts = $this->getContainerStatements($fileObject);
```
Comment (nicodh) [inline Classes/Parser/NodeFactory.php:114]: This resulted in double generated class code in each file when I tested it (and that's the error in CI Unit Tests).
I think we have to add a flag to the FileObject whether or not to add the declare strict line...

**Comment by nicodh** [comment]
I would suggest to add a boolean setting flag declareStrictTypes which defaults to true

**Comment by liayn** [comment]
Yes, I scuttled around here a bit with various implementation variants.
The thing is, we must not put the strict-stuff in the ext_*.php files.

That's why I had a look at the parser and tried to really read the declare stmt from the template. But in case of the namespace-tests this does not work out, as the outer stmts (outside of namespace) are not rendered anymore.

**Pull request merged**